lxqt-session/src/lxqtmodman.cpp

```diff
@@ -83,7 +83,8 @@ void LXQtModuleManager::startup(LXQt::Settings& s)
     startConfUpdate();
 
     // Start window manager
-    startWm(&s);
+    if(QX11Info::isPlatformX11())
+        startWm(&s);
```

Comment:
`QGuiApplication::platformName()` works here too, as you did below; no need to `QX11Info`.

Reply:
Ok.